src/redteam/plugins/harmful.ts

```diff
@@ -191,7 +191,7 @@ class HarmfulPlugin extends PluginBase {
     return [
       {
         type: `promptfoo:redteam:${this.category.key}`,
-        metric: `${this.category.key}`,
+        metric: 'Harmful',
```

Comment:
Can we add test coverage for this so it doesn't happen again?

Reply:
yes but I have a question about this test.  can we address this privately